Domain Analysis: Deep Dive into markettrendingcenter.com Historical and Current Data

abdarafi opened the pull request:
This PR provides an in-depth analysis of the domain markettrendingcenter.com (Lazarus Group's related domain) by employing various domain and web exploration tools:

Files Added:

1. `markettrendingcenter.com-namedroppers-search-result.png`
Results from the NameDroppers domain search.
2. `markettrendingcenter.com-whoisology-search-result.png`
 WHOIS data capture from Whoisology.
4. `markettrendingcenter.com-crtsh-search-result.png`
Certificate transparency logs from crt.sh.
6. `markettrendingcenter.com-web-archive-search-result.png`
An interesting snapshot from the Web Archive dated Dec 16, 2021, that reveals a disclosed Microsoft IIS hyperlink.
8. `markettrendingcenter.com-ctfr-result.png`
Subdomain findings using the CTFR tool.
10. `markettrendingcenter.com-cloudflair-result.png`
Potential origin IPs using Cloudflair.
12. `M11109816xM11115825-kestrel-analysis.png`
Insights from our analysis using the Kestrel threat hunting language.

cc: @VStev 

### Checks
- [x] Your YAML file has required fields as listed in [Specs](https://github.com/ntust-im-labyrinth/cybersecurity-threat-research/tree/f63bd214302765c31fbffbfe5b46744d0f8f6877/spec)
- [x] All checks have passed [yaml-lint passed](https://github.com/ntust-im-labyrinth/cybersecurity-threat-research/actions/workflows/lint.yaml)
- [x] You agree to contribute your work under [MIT license](https://github.com/ntust-im-labyrinth/cybersecurity-threat-research/blob/f63bd214302765c31fbffbfe5b46744d0f8f6877/LICENSE)

VStev commented:
Added another file that contains the complete huntbook outputs for better analysis:

> markettrendingcenter.com-log4shell-kestrel-analysis..html

abdarafi commented:
We also renamed and moved our previous screenshot of the kestrel installation to the correct location to align with the course material's instruction for kestrel learning evidence screenshot location.

new location: `threat/hunting/M11109816xM11115825-kestrel-install.png`

cc: @VStev
